logstash-core/src/main/java/org/logstash/log/SlowLoggerExt.java

```diff
@@ -55,17 +55,29 @@ public SlowLoggerExt(final Ruby runtime, final RubyClass metaClass) {
         super(runtime, metaClass);
     }
 
+    SlowLoggerExt(final Ruby runtime, final RubyClass metaClass, final String loggerName,
```

Comment:
The mix of java classes and `IRubyObject` in this constructor feels a bit "off" to me. Can/should we push the casting/converting up to the caller here?

Reply:
wanted that initially but than would need to do the same `((RubyNumeric) args[1]).getLongValue()`
from 2 different places, so that is why I left it with `IRubyObject`, still a good point to do it properly ...